Yoast/wordpress-seo
# Bring config wizard question in line with 7.0 questions

## Summary

This PR can be summarized in the following changelog entry:

* Removes media post type from the config wizard, brings the question about indexing in line with the rest of the plugin.

## Test instructions

This PR can be tested by following these steps:

* Open the configuration wizard, see the search engine visibility section.

## Quality assurance


Fixes #8677
